code/game/objects/items/weapons/cards_ids.dm

```diff
@@ -392,12 +392,20 @@ var/const/NO_EMAG_ACT = -50
 	access = get_all_station_access() | get_all_centcom_access()
 	..()
 
+/obj/item/card/id/command
+	desc = "A high-tech holocard displaying the credentials of a Command member."
+	icon_state = "command_card"
+/obj/item/card/id/command/captain
+	desc = "A high-tech holocard displaying the commanding credentials of a ship captain."
+	icon_state = "captain_card"
+/obj/item/card/id/command/bridge
+	desc = "A high-tech holocard displaying the lowly credentials of a Bridge Crewman."
+	icon_state = "bridge_card"
+
```

Comment:
- need spacing between these definitions
- these are specific to SCC so they should have a path like `/obj/item/card/id/scc/...`
- there should be generic versions of these so they can be used outside of horizon (for other SCC facilities) (by versions I mean separate type paths and descs). for example:
  - `/obj/item/card/id/scc` with generic desc
  - `/obj/item/card/id/scc/gold` with generic desc
  - `/obj/item/card/id/scc/silver` with generic desc
  - `/obj/item/card/id/scc/gold/captain` specifically for captains
  - etc

Reply:
Changes made. Wanted to try and keep unique descriptions for each, so hopefully this works. HRA gets the non-captain version of the gold card. Lmk if those work